.github/workflows/get-translations.yml

```diff
@@ -31,17 +37,42 @@ jobs:
       - name: Fetch latest dev
         run: git fetch origin dev
 
-      - name: Get translations
-        run: npx ts-node -O '{"module":"commonjs"}' ./src/scripts/crowdin/translations/getTranslations.ts
-        env:
-          CROWDIN_API_KEY: ${{ secrets.CROWDIN_API_KEY }}
-          CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID }}
-
       - name: Authenticate GitHub CLI
         run: |
           echo ${{ secrets.GITHUB_TOKEN }} | gh auth login --with-token
 
+      # Build translations
+      - name: Build Crowdin project
+        id: build-crowdin
+        run: |
+          OUTPUT=$(npx ts-node -O '{"module":"commonjs"}' ./src/scripts/crowdin/translations/triggerBuild.ts)
+          echo "$OUTPUT"
+        shell: bash
+
+      - name: Await latest build to finish
+        env:
+          BUILD_ID: ${{ steps.build-crowdin.outputs.buildId}}
+        run: |
+          OUTPUT=$(npx ts-node -O '{"module":"commonjs"}' ./src/scripts/crowdin/translations/awaitLatestBuild.ts)
+          echo "$OUTPUT"
```

Comment:
Similar syntax to above, but this time we're outputting `::set-output name=buildSuccess::true` (or false) depending if the build eventually switched to "finished" status or not.